PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -489,6 +574,440 @@ struct HfCandidateCreatorXic0Omegac0 {
+                       kfOmegac0Candidate.cosPaV0ToCasc, kfOmegac0Candidate.cosPaOmegacToPv, kfOmegac0Candidate.cosPaCascToOmegac, kfOmegac0Candidate.cosPaXYV0ToCasc, kfOmegac0Candidate.cosPaXYOmegacToPv, kfOmegac0Candidate.cosPaXYCascToOmegac,
+                       kfOmegac0Candidate.ctOmegac, kfOmegac0Candidate.ctCasc, kfOmegac0Candidate.ctV0,
+                       pseudorapV0Dau0, pseudorapV0Dau1, pseudorapCascBachelor, pseudorapCharmBachelor,
+                       KFOmegaC0.GetEta(), KFOmega.GetEta(), KFV0.GetEta(),
+                       dcaxyV0Dau0, dcaxyV0Dau1, dcaxyCascBachelor,
+                       dcazV0Dau0, dcazV0Dau1, dcazCascBachelor,
+                       kfOmegac0Candidate.kfDcaCascDau, kfOmegac0Candidate.kfDcaV0Dau, kfOmegac0Candidate.kfDcaOmegacDau,
+                       decLenCharmBaryon, decLenCascade, decLenV0, errorDecayLengthCharmBaryon, errorDecayLengthXYCharmBaryon);
+      // fill kf table
+      kfCandidateData(kfOmegac0Candidate.nSigmaTPCPiFromOmegac, kfOmegac0Candidate.nSigmaTOFPiFromOmegac,
+                      kfOmegac0Candidate.nSigmaTOFKaFromCasc, kfOmegac0Candidate.nSigmaTOFKaFromCasc,
+                      kfOmegac0Candidate.nSigmaTPCPiFromV0, kfOmegac0Candidate.nSigmaTPCPrFromV0,
+                      kfOmegac0Candidate.kfDcaXYPiFromOmegac, kfOmegac0Candidate.kfDcaCascDau, kfOmegac0Candidate.kfDcaOmegacDau, kfOmegac0Candidate.kfDcaXYCascToPv,
+                      kfOmegac0Candidate.chi2GeoV0, kfOmegac0Candidate.chi2GeoCasc, kfOmegac0Candidate.chi2GeoOmegac, kfOmegac0Candidate.chi2MassV0, kfOmegac0Candidate.chi2MassCasc,
+                      kfOmegac0Candidate.ldlV0, kfOmegac0Candidate.ldlCasc, kfOmegac0Candidate.ldlOmegac,
+                      kfOmegac0Candidate.chi2TopoV0ToPv, kfOmegac0Candidate.chi2TopoCascToPv, kfOmegac0Candidate.chi2TopoPiFromOmegacToPv, kfOmegac0Candidate.chi2TopoOmegacToPv,
+                      kfOmegac0Candidate.chi2TopoV0ToCasc, kfOmegac0Candidate.chi2TopoCascToOmegac,
+                      kfOmegac0Candidate.decayLenXYLambda, kfOmegac0Candidate.decayLenXYCasc, kfOmegac0Candidate.decayLenXYOmegac,
+                      kfOmegac0Candidate.cosPaV0ToCasc, kfOmegac0Candidate.cosPaV0ToPv, kfOmegac0Candidate.cosPaCascToOmegac, kfOmegac0Candidate.cosPaCascToPv, kfOmegac0Candidate.cosPaOmegacToPv,
+                      kfOmegac0Candidate.massV0, kfOmegac0Candidate.massCasc, kfOmegac0Candidate.massOmegac,
+                      kfOmegac0Candidate.rapOmegac, kfOmegac0Candidate.ptPiFromOmegac, kfOmegac0Candidate.ptOmegac,
+                      kfOmegac0Candidate.cosThetaStarPiFromOmegac,
+                      kfOmegac0Candidate.ctOmegac,
+                      kfOmegac0Candidate.etaOmegac,
+                      v0NDF, cascNDF, charmbaryonNDF, v0NDF_m, cascNDF_m,
+                      v0Chi2OverNdf, cascChi2OverNdf, charmbaryonChi2OverNdf, v0Chi2OverNdf_m, cascChi2OverNdf_m);
```

Comment:
You have several variables that are duplicated in the two tables, such as `kfOmegac0Candidate.cosPaV0ToCasc`. Please do not duplicate them (they increase the memory for no reason), see also my comment above.

Reply:
Done.